test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.Protocol.cs

```diff
@@ -565,6 +565,62 @@ public class Protocol
                     await connection.DisposeAsync().OrTimeout();
                 }
             }
+
+            [Fact]
+            public async Task ClientPingsAutomatically()
+            {
+                var connection = new TestConnection();
+                var hubConnection = CreateHubConnection(connection);
+
+                hubConnection.TickRate = TimeSpan.FromMilliseconds(30);
+                hubConnection.PingInterval = TimeSpan.FromMilliseconds(80);
+
+                try
+                {
+                    await hubConnection.StartAsync().OrTimeout();
+                    await Task.Delay(120);
+
+                    var received = await connection.ReadSentTextMessageAsync().OrTimeout(TimeSpan.FromMilliseconds(100));
```

Comment:
nit: `var sent`

Reply:
Make it `OrTimeout(TimeSpan.FromSeconds(1))`

It's hard to write tests for small wait durations, so we try to err on the conservative side. For this test we know we'll get a ping before 1 second, if we don't, something went wrong and we'll timeout